src/main/webapp/resources/js/pages/projects/processing/AnalysisTemplates.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+  deleteAnalysisTemplateForProject,
+  fetchAnalysisTemplatesForProject,
+} from "../../../apis/projects/settings";
+import { Button, List, notification, Popconfirm, Tag } from "antd";
+import { IconRemove } from "../../../components/icons/Icons";
+
+export function AnalysisTemplates({ projectId, canManage }) {
+  const [templates, setTemplates] = React.useState();
+
+  React.useEffect(() => {
+    fetchAnalysisTemplatesForProject(projectId).then((data) =>
+      setTemplates(data)
+    );
+  }, []);
+
+  const removeAutomatedPipeline = (template) => {
+    deleteAnalysisTemplateForProject(template.id, projectId).then((message) => {
+      notification.success({ message });
+      const filteredTemplates = templates.filter((t) => t.id !== template.id);
+      setTemplates(filteredTemplates);
+    });
```

Comment:
If there is an error for whatever reason deleting the template the user should get a notification that it failed. Thoughts?

Reply:
I don't really think so, there has already been a check to make sure the user can manage them, and the user cannot modify the identifier so I don't think this would be the case.

Reply:
Sounds good